js/twister_following.js

```diff
@@ -353,6 +353,8 @@ function updateFollowing(cbFunc, cbArg) {
 function follow(user, publicFollow, cbFunc, cbArg) {
     if( followingUsers.indexOf(user) < 0 ) {
         followingUsers.push(user);
+        twisterFollowingO.update(user);
```

Comment:
looks like it covers #195 and another issue that I had previously: I had same account on two nodes with [different](https://twisterio.com/post/tasty/956) number of following users. now it's gone.